README.md

```diff
@@ -2,410 +2,17 @@
 
 The Cloud Native Computing Foundation (CNCF) participates in a variety of mentoring programs. CNCF is a great place to spend a summer learning, coding, participating, and contributing. We are an exciting open source foundation with a vibrant community of projects, and we look forward to your application and your project ideas!
 
-- [CNCF Mentoring Initiatives](#cncf-mentoring-initiatives)
-  - [Organization Admins](#organization-admins)
-  - [Communication](#communication)
-  - [Success Stories](#success-stories)
-  - [Mentoring Programs](#mentoring-programs)
-  - [Program Statistics](#program-statistics)
-    - [LFX (ex-Community Bridge)](#lfx-ex-community-bridge)
-      - [2022](#2022)
-        - [Summer](#summer)
-        - [Spring](#spring)
-      - [2021](#2021)
-        - [Fall](#fall)
-        - [Summer](#summer)
-        - [Spring](#spring)
-      - [2020](#2020)
-        - [Q3-Q4](#q3-q4)
-        - [Q2](#q2)
-        - [Q1](#q1)
-      - [2019](#2019)
-        - [Pilot](#pilot)
-    - [GSoC](#gsoc)
-      - [2020](#2020-1)
-      - [2019](#2019-1)
-      - [2018](#2018)
-      - [2017](#2017)
-  - [Code of Conduct](#code-of-conduct)
-
-## Organization Admins
-
-If you need help with anything mentoring at CNCF, you can file an issue at this repo or reach out to us using the [Discussions boards](https://github.com/cncf/mentoring/discussions).
-
-Organization admins for specific mentorship programs are listed on the program's respective pages.
-
-## Communication
-
-Please reach out to us using the [Discussions boards](https://github.com/cncf/mentoring/discussions).
-
-We are also available on the [CNCF slack](https://slack.cncf.io/). Please don't use DMs unless strictly necessary as doing so both has the potential of overwhelming project maintainers and others with similar questions lose the benefit of public discussion.
-
-It's best if you use a public communication channel whenever possible; however, if you need to communicate in private, please feel free to send the admins a note via mentoring@cncf.io (please use the public channels for any project-related discussion).
```

Comment:
Not sure if this kind of general information is removed temporarily from the main README, but it is a good idea to keep some general information here. Links to mentor/mentee/orgAdmin guides, links to communication document, etc.

Reply:
We moved this section over here:
https://github.com/cncf/mentoring/blob/v2/mentoring-wg/communications.md

Might add a link on the main README.md and not duplicate the information.
WDYT?

Reply:
+1 on having a link (or multiple links for multiple docs)

Reply:
@nate-double-u , Added a more information and links in the readme via https://github.com/cncf/mentoring/pull/769